README.md

```diff
@@ -4,4 +4,47 @@ Github Actions used in projects of the WordPress team.
 
 - PHPStan
 - Dependabot (auto-merge)
-- 
+- Markdown Lint
```

Comment:
Deze heb je net verwijderd?

Reply:
Nee alleen de config, als je hem bijvoorbeeld in de [skeleton-package](https://github.com/yardinternet/skeleton-package) aanroept pakt hij de config uit de root daar